lib/cext/ruby.h

```diff
@@ -101,11 +101,11 @@ VALUE rb_tr_hidden_variable_get(VALUE object, const char *name);
 
 // Handles
 
-void *rb_tr_handle_for_managed(void *managed);
-void *rb_tr_handle_for_managed_leaking(void *managed);
-VALUE rb_tr_managed_from_handle_or_null(void *handle);
-void *rb_tr_managed_from_handle(void *handle);
-void rb_tr_release_handle(void *handle);
+void* rb_tr_handle_for_managed(VALUE managed);
+void* rb_tr_handle_for_managed_leaking(VALUE managed);
```

Comment:
I find
```c
void *rb_tr_handle_for_managed(VALUE managed);
```
much less clear on being a conversion function. But it is inconsistent, indeed.